prefixy.js

```diff
@@ -190,6 +191,165 @@ class Prefixy {
 
     return this.client.batch(commands).execAsync();
   }
+
+  // Disk Persistence Methods
+
+  async loadPrefixFromDisk(prefix) {
+    await this.client.zremrangebyrank(prefix, 0, -1);
```

Comment:
potentially dont need this extra round trip since redis sorted sets take care of duplicates